sex_offender_search.go

```diff
@@ -0,0 +1,32 @@
+package checkr
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// SexOffenderSearch ...
+// https://docs.checkr.com/#tag/Sex-Offender-Registry-Search
+type SexOffenderSearch struct {
+	ID             string    `json:"id"`
+	Object         string    `json:"object"`
+	URI            string    `json:"uri"`
+	Status         string    `json:"status"`
+	CreatedAt      time.Time `json:"created_at"`
+	CompletedAt    time.Time `json:"completed_at"`
+	TurnaroundTime int       `json:"turnaround_time"`
+	Records        []struct {
+		Registry          string `json:"registry"`
+		FullName          string `json:"full_name"`
+		Age               int    `json:"age"`
+		Dob               string `json:"dob"`
```

Comment:
Make this into `*time.Time` as well. Also, I did `dob` as string on `candidates.go`. Let's update that to `*time.Time` too.

Reply:
hi @magicalbanana  i tried doing dob to *time.Time , later i parsed a time and sent to createcandidate request i keep getting time stamp parsing errors from checkr. seems there are some open issues opened in go lang for it as well. so i falled back to string for DOB while creating candidate